cybersecuritytools/csls/cli.py

```diff
@@ -0,0 +1,15 @@
+import click
+
+from .generate_cloudwatch_logs.cli import generate_cloudwatch_logs
+from .hec_index_checker.cli import check_hec_token
```

Comment:
Why were relative imports needed here? Will that work when deployed? I'm always nervous when I see them!

Reply:
The were mainly there as I'm not confident the directory structure won't change as we add more tools into this repo and they are more flexible for restructuring than absolute inputs.

Reply:
Cool, you know what you're doing :+1: